module/platform/Webots/src/Webots.cpp

```diff
@@ -408,6 +405,9 @@ namespace module::platform {
                             buffer.erase(buffer.begin(), std::next(buffer.begin(), sizeof(length) + length));
                         }
                     }
+
+                    // Service the watchdog
+                    emit<Scope::WATCHDOG>(ServiceWatchdog<Webots>());
```

Comment:
This is also still in the wrong place, should be on line 403ish

Reply:
This was fixed here :( https://github.com/NUbots/NUbots/pull/475/commits/3a807992b4ef901582def9316930d79540786ac8

Reply:
Is this all good now?